scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -53,26 +54,41 @@ def generate_simple_geojsons(self, verbose=True):
                      tempfile.TemporaryFile(mode='r+') as f2:
 
                     geojson.dump(coords, f1)
-                    geojson.dump(
-                        self.simplifier.simplify(coords,
-                                                 verbose=verbose,
-                                                 epsilon=0.05), f2)
+                    simple = self.simplifier.simplify(coords,
+                                                      verbose=verbose,
+                                                      epsilon=0.01)
+                    geojson.dump(simple, f2)
 
                     # Rewind files to start for reading.
                     f1.seek(0)
                     f2.seek(0)
 
                     plotter.compare_plots(gpd.read_file(f1),
                                           gpd.read_file(f2),
-                                          show=False)
+                                          show=show)
+                    self.simple_geojsons[geoid] = simple
             except AssertionError:
                 logging.error("Simplifier failure on GeoJSON below:\n", coords)
         plt.show()
 
-    # def generate_mcf(self, path='low_res_geojsons.mcf'):
-    #     with open(path, 'w') as f:
+    def generate_mcf(self, path='low_res_geojsons.mcf'):
+        temp = (
+            "Node: dcid:{geoid}\n"
+            "typeOf: dcs:{type}\n"
+            "geoJsonCoordinatesLowRes: {coords_str}\n\n"
+        )
```

Comment:
```suggestion
        temp = "\n".join([
            "Node: dcid:{geoid}",
            "typeOf: dcs:{type}",
            "geoJsonCoordinatesLowRes: {coords_str}\n"
        ])
```

I am not sure this is an improvement.  It does mean that the programmer need not worry about forgetting to add a newline.

Triple quotes section is an alternative:
temp = '''
Node: dcid:{geoid}
typeOf: dcs:{type}
geoJsonCoordinatesLowRes: {coords_str}

'''
That might be better.

Reply:
Changed to `.join` approach. Triple quotes leave a weird indent in all lines after the first.